### ISSUE TITLE
Feature Request: quickMove, quickCopy, and quickJump from the Current Folder toolbar

### ISSUE THREAD

**Issue by stw203** [RealRaven2000/QuickFolders#347]
It would be much more convenient to have the ability (if possible) to drag/drop emails to the center of the three-pane screen rather than dragging all the way up to the QF menu bar. Thanks!

**Comment by RealRaven2000** [comment]
> It would be much more convenient to have the ability (if possible) to drag/drop emails to the center of the three-pane screen rather than dragging all the way up to the QF menu bar. Thanks!

I agree! I could add a quickMove target button to the "Current Folder bar" - however I would like to keep the textbox at the top left if that's ok? Also, if you have a Pro license you can configure keyboard shortcuts to use this feature:

![image](https://user-images.githubusercontent.com/6314674/224391420-31fd0e66-525b-4831-a2f4-a62db6a87e8f.png)

**Comment by stw203** [comment]
Thanks for the keyboard tip! I will set that up too!

I think that keeping the text box where it is would be fine...as long as the cursor jumps there when the email(s) get dropped on the target button...regardless of whether it's the original/current one or the proposed new one on the Current Folder bar.

Thanks for the quick reply!

**Comment by crablouis** [comment]
Having the QuickJump drag'n'drop target on the QuickTab bar for all or some categories would add utility, I think. However, I think it's important to keep the target for this also in the same relative location for all categories. 

Some of my categories have MANY tabs. Some have just few. So if the QuickJump target is at, say, the last position on the bottom right QuickTab row, it'll jump around as we change categories. 

It could go immediately next to the current Recent Folders tab, but that's already right next to the current category switcher, options, reading list, and QuickJump target.

**Comment by RealRaven2000** [comment]
No I meant here - the current folder bar is in the center of the screen:

![image](https://user-images.githubusercontent.com/6314674/224425691-dae5c611-8eb0-42e1-b666-6dffcedbd9b5.png)

Also you can actually drag the envelope on the left (it represents the selected mails) - to any tab or even onto the recent folder clock right beside it. We could put the quickMove button just in between. It makes _loads_ of sense as the current folder toolbar is actually all about navigation (I was thinking of rebranding it the "Navigation Toolbar").

The only problem with it is the amount of vertical space it takes up in the classic 3 pane view I haven't really found a better solution for that though.

**Comment by crablouis** [comment]
Ah... the Current Folder Bar, which I long ago disabled. I found that the space taken didn't afford enough added utility. afaik, I prefer to just drag from the message listing. Others, of course, may prefer differently. Carry on.

**Comment by RealRaven2000** [comment]
> Ah... the Current Folder Bar, which I long ago disabled. I found that the space taken didn't afford enough added utility. afaik, I prefer to just drag from the message listing. Others, of course, may prefer differently. Carry on.

I wish I could find a way to make it take up less (vertical) space. The location is quite crucial it is in the center of screen right between mails and mail preview, so it's the least amount of screen travel for the mouse.

**Comment by RealRaven2000** [comment]
... and if you are usually working with the latest mails at the end of the list, it is very close as a drop target:

![image](https://user-images.githubusercontent.com/6314674/224455949-7891883c-40b7-490a-b07a-fea68170a9e0.png)

**Comment by RealRaven2000** [comment]
> It would be much more convenient to have the ability (if possible) to drag/drop emails to the center of the three-pane screen rather than dragging all the way up to the QF menu bar. Thanks!

where would it (ideally) be if using the 3 column view?
